blocks/header/header.css

```diff
@@ -32,7 +32,7 @@ header .nav[aria-expanded='false'] .nav-hamburger-icon::before {
   width: 20px;
   height: 2px;
   border-radius: 3px;
-  background: currentColor;
+  background: currentcolor;
```

Comment:
Isn't it `currentColor`? https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword

Reply:
i also thought that, but the css linter complained. let's make sure the linter is configured correctly...

Reply:
According to https://stylelint.io/user-guide/rules/list/value-keyword-case/ all keyword values are expected to be lowercase, referring to https://www.w3.org/TR/css-color-4/#currentcolor-color.

I'd say we stand corrected 😄